xcell/mappers/mapper_CatWISE.py

```diff
@@ -37,17 +40,39 @@ def get_catalog(self):
                  self.config.get('flux_max_W1', 16.4))]
         return self.cat_data
 
+    # Correction to Density
+    def get_ecliptic_correction(self):
```

Comment:
I believe it would be good if we make get_eclipttic_correction a private method.
`get_ecliptic_correction(self) --> _get_ecliptic_correction(self)`